### DISCUSSION TITLE
(3.3.0) Taxonomies have a broken slug field

### DISCUSSION THREAD

**Discussion by eugis42** [statamic/cms#5532] [Troubleshooting] [answered]
Hello there,

I'm experiencing a problem on my current Statamic install as well as fresh new installs (3.3.0) where when I create or edit taxonomies, the slug field is not visible, it doesn't generate and can't be removed and replaced in the blueprint. Any idea of what I'm doing wrong? 

Thanks a lot

<img width="1243" alt="Screenshot 2022-03-16 at 17 45 56" src="https://user-images.githubusercontent.com/7288560/158643674-2db43f6d-9408-4afc-b8f0-c074fb8c6ca3.png">
i

**Comment by jasonvarga** [comment, marked as answer]
Duplicate of #5520 and #5522 

It's already fixed and will be in 3.3.1, which will be released in a moment.

**Reply by eugis42** [reply]
Oh thanks a lot and sorry for the duplicate.
